### PR TITLE
Update Density Mask Texture import doc

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
Fix for https://fogbugz.unity3d.com/f/cases/1268690/

Removed the Density Volume Texture Tool because it is replaced by the new 3D Texture Importer.
Updated the doc on how to create a Density Mask Texture accordingly.

Also fixed an unrelated bug where modifying the Mask Texture with an external tool would not reload it in the Density Volume.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Density-Volume.md [hunk 1]
```diff
@@ -34,16 +34,15 @@ To create a Density Volume, right click in the Hierarchy and select __Rendering
 
 ## Creating a Density Mask Texture
 
-1. In image-editing software of your choice, prepare a grayscale 2D texture with values between 0 and 1 and of size 1024x32. This size describes a 3D texture of size 32x32x32 with 32 slices laid out one after another.
-2. Import this texture into the Unity Editor and set the Texture Import Settings:
+1. In image-editing software of your choice, prepare a grayscale flipbook texture and [import it as a 3D texture](https://docs.unity3d.com/2020.2/Documentation/Manual/class-Texture3D.html). For example, a texture of size 1024x32 describes a 3D texture of size 32x32x32 with 32 slices laid out one after another.
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Documentation~/Density-Volume.md:37]: @adrien-de-tocqueville can you also update the upgrade guide 2020.1 to 2020.2 to explain that we have remove this tool and we should use the regular importer? Thanks

Comment (JordanL8) [reply com.unity.render-pipelines.high-definition/Documentation~/Density-Volume.md:37]: The rest of the doc looks good. @adrien-de-tocqueville, can you ping me when you've added the entry to the upgrade guide so I can have a look at that?

Comment (adrien-de-tocqueville) [reply com.unity.render-pipelines.high-definition/Documentation~/Density-Volume.md:37]: @JordanL8 done

File: com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md [hunk 2]
```diff
@@ -165,3 +165,8 @@ With:
 ```
 protected override void Execute(CustomPassContext ctx) { ... }
 ```
+
+## Density Volume Mask Texture
+
+Previously, Density Mask Textures needed the __Density Volume Texture Tool__ in the __Window > Rendering__ menu to be converted from a 2D flipbook format to the 3D format required by the Density Volume Component.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md:171]: Previously, to convert a 2D flipbook texture to the 3D format that Density Mask Textures require, you needed to use the __Density Volume Texture Tool__ in the __Window > Rendering__ menu.

File: com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md [hunk 3]
```diff
@@ -165,3 +165,8 @@ With:
...
+With Unity 20202.2, this can be done directly through the __Texture Importer__. To see the detailed steps on how to use the importer, refer to the [Density Volume documentation](Density-Volume.md).
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md:172]: From Unity 2020.2, you can now do this conversion directly through the __Texture Importer__. For information on how to use the importer, see to the [Density Volume documentation](Density-Volume.md).

**Review by JordanL8 — CHANGES_REQUESTED** [review]
Added a couple of suggestions.

**Review by JordanL8 — APPROVED** [review]

**Pull request merged**